code/__DEFINES/MC.dm

```diff
@@ -0,0 +1,78 @@
+#define MC_TICK_CHECK ( ( TICK_USAGE > Master.current_ticklimit || src.state != SS_RUNNING ) ? pause() : 0 )
```

Comment:
```suggestion
#define MC_TICK_CHECK ( ( TICK_USAGE > Master.current_ticklimit || state != SS_RUNNING ) ? pause() : 0 )
```

Reply:
This is fine and good as is, `src.` is safer in macros when someone may have named a var the same thing.